### PR TITLE
fix: dashboard X spacing to be consistent with headers

### PR DESCRIPTION
**Pull request by andrewdw**
Closes: #12528 

### Description:
- makes dashboard spacing consistent with headers

CleanShot 2024-11-20 at 16 23 32

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Review by magnew — APPROVED** [review]
Tested with and without tabs, looks good ✅ 
Code makes sense if we want tabs going to the edges, which it looks like we do ✅

File: packages/frontend/src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -692,12 +692,7 @@ const Dashboard: FC = () => {
                 <Group position="apart" align="flex-start" noWrap px={'lg'}>
                     <Box style={{ flexGrow: 1, overflow: 'auto' }}>
                         {/* This Group will take up remaining space (and not push DateZoom) */}
-                        <Group
-                            position="apart"
-                            align="flex-start"
-                            noWrap
-                            px={'lg'}
-                        >
+                        <Group position="apart" align="flex-start" noWrap>
```
Comment (joaoviana) [inline packages/frontend/src/pages/Dashboard.tsx:695]: I believe you can remove the Box above and just have the group like this

```
<Group
                        position="apart"
                        align="flex-start"
                        noWrap
                        grow
                        sx={{
                            overflow: 'auto',
                        }}
                    >


```

**Review by joaoviana — CHANGES_REQUESTED** [review]
Once PR is rebased, let's wait for the e2es to pass 👍

**Review by joaoviana — APPROVED** [review]
shipit

**Pull request merged**